runtime/launcher.m

```diff
@@ -357,6 +358,8 @@
 		// So route through MONO_DEBUG
 		setenv ("MONO_DEBUG", "no-gdb-backtrace", 0);
 	}
+
+	setenv ("MONO_CFG_DIR", [monobundle_dir UTF8String], 1);
```

Comment:
We already call `mono_set_dirs` [1], whose second parameter sets the config directory. Maybe we should check why that doesn't work instead?

[1] https://github.com/xamarin/xamarin-macios/blob/master/runtime/launcher.m#L495-L499